CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Valhalla `/expansion` examples in the Jupyter Notebook
 
+## Fixed
```

Comment:
```suggestion
### Fixed
```

Reply:
Arghhh! No more mistakes!
Let me just squash the commits so there is no noise and it should be ok now.